raiden/ui/cli.py

```diff
@@ -840,9 +841,12 @@ def print_step(description: str, error: bool = False) -> None:
             port = next(free_port_generator)
 
             args["api_address"] = f"localhost:{port}"
-            args["config"] = deepcopy(App.DEFAULT_CONFIG)
             args["environment_type"] = environment_type
-            args["extra_config"] = {"transport": {"available_servers": server_urls}}
+
+            # Matrix server
+            # TODO: do we need more than one here?
```

Comment:
This is only smoketest code, right? I guess we don't. @ulope can you confirm so that the comment can be removed?